src/views/ModelPlan/TaskList/ITSolutions/_components/NeedQuestionAndAnswer/index.tsx

```diff
@@ -123,19 +79,19 @@ type NeedQuestionAndAnswerProps = {
   modelID: string;
   expanded?: boolean;
   solution?: GetOperationalSolutionType; // Solution passed as prop if want to render a SolutionCard beneath the need question
+  isRenderingOnSolutionsDetails?: boolean;
 };
 
 const NeedQuestionAndAnswer = ({
   className,
   operationalNeedID,
   modelID,
   expanded,
-  solution
+  solution,
+  isRenderingOnSolutionsDetails = false
 }: NeedQuestionAndAnswerProps) => {
   const { t } = useTranslation('itSolutions');
-
-  // Toggle the collapsed state of operational need question/answer
-  const [infoToggle, setInfoToggle] = useState<boolean>(false);
+  const [isModalOpen, setIsModalOpen] = useState(false);
 
   // Fetch operational need answer to question
   const { data: need } = useQuery<
```

Comment:
This query is causing the cache to interfere with another query on the same page, as this variable isn't being passed through

`includeNotNeeded: false`

To reproduce this issue, go to the select solutions page and only have one solution selected, then continue and see that all the solution are listed on the solution implementation details page